docs/src/faq/another-logging-lib.md

```diff
@@ -1,4 +1,4 @@
-# Another logging library?
+# [Another logging library?](@id yall)
```

Comment:
What is `yall` ?

Reply:
yet another logging library, I don't really care what the id is called as long as I can use it